### PR TITLE
chore(core): updates to nx wrapper and naming

### PR DESCRIPTION
**Pull request by AgentEnder**
## Related Issue(s)

Fixes #

### CODE REVIEW

**Review by FrozenPandaz — APPROVED** [review]

File: docs/shared/concepts/more-concepts.md [hunk 1]
```diff
@@ -14,4 +14,4 @@
 - [Grouping Libraries](/more-concepts/grouping-libraries)
 - [Buildable and Publishable Libraries](/more-concepts/buildable-and-publishable-libraries)
 - [How the Project Graph is Built](/more-concepts/how-project-graph-is-built)
-- [Encapsulated Nx and the Nx Wrapper](/more-concepts/encapsulated-nx-and-the-wrapper)
```
Comment (bcabanes) [inline docs/shared/concepts/more-concepts.md:17]: Do we need a redirect rule for this?

Comment (AgentEnder) [reply docs/shared/concepts/more-concepts.md:17]: No, page hasn't been linked to enough for that to be necessary

Comment (AgentEnder) [reply docs/shared/concepts/more-concepts.md:17]: No published content, etc

Comment (FrozenPandaz) [reply docs/shared/concepts/more-concepts.md:17]: This one we're fine. It's pretty new and we haven't really talked about it yet.

**Review by mandarini — APPROVED** [review]

**Review by bcabanes — APPROVED** [review]

**Pull request merged**